numpy/core/tests/test_fromnumeric.py

```diff
@@ -1,17 +0,0 @@
-from __future__ import division, absolute_import, print_function
```

Comment:
I added that file myself from #7000 that served as a stopgap measure to test that ```put``` would not accept not ```ndarray``` args.  Also, a lot of the tests of ```fromnumeric.py``` functions are located elsewhere in the tests.